tests/test_series.py

```diff
@@ -1591,3 +1591,15 @@ def test_updated_astype() -> None:
         pd.Series,
         Decimal,
     )
+
+    # Categorical
+    check(
+        assert_type(s.astype(pd.CategoricalDtype()), "pd.Series[Any]"),
+        pd.Series,
+        pd.CategoricalDtype,
+    )
+    check(
+        assert_type(s.astype("category"), "pd.Series[Any]"),
+        pd.Series,
+        pd.CategoricalDtype,
```

Comment:
for both of these tests, the last argument to `check()` should be `np.integer` because that is the type of the first element of the corresponding `Series`.